src/library/Uno.Material/Styles/Controls/ComboBox.xaml

```diff
@@ -27,16 +27,16 @@
 					ResourceKey="OnSurfaceFocusedBrush" />
 
 	<StaticResource x:Key="ComboBoxArrowForegroundThemeBrush"
-					ResourceKey="OnSecondaryMediumBrush" />
+					ResourceKey="OnBackgroundBrush" />
 
-	<SolidColorBrush x:Key="ComboBoxForegroundThemeBrush"
-					 Color="{ThemeResource OnSecondaryColor}" />
+	<StaticResource x:Key="ComboBoxForegroundThemeBrush"
+					ResourceKey="OnBackgroundBrush" />
 
-	<StaticResource x:Key="ComboBoxFocusedBackgroundThemeBrush"
-					ResourceKey="OnSecondaryMediumBrush" />
+	<SolidColorBrush x:Key="ComboBoxPlaceholderTextForegroundThemeBrush"
+					 Color="{ThemeResource OnBackgroundColor}"
+					 Opacity="0.74" />
 
-	<StaticResource x:Key="ComboBoxPlaceholderTextForegroundThemeBrush"
-					ResourceKey="OnSecondaryMediumBrush" />
+	<CornerRadius x:Key="BorderRadius">4</CornerRadius>
```

Comment:
That's a too general name. Use the VariableNames in the zeplin.
https://app.zeplin.io/project/5e344ca7405a53a6ef303bc1/screen/5e4d88eb8a67138239e14783

Reply:
Now it's DefaultComboBoxBorderRadius.